content/en/Integrations/_index.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "Cobalt Integrations"
+linkTitle: "Integrations"
+weight: 110
+description: >
+  Integrate third-party apps and configure webhooks.
+---
+
+{{% pageinfo %}}
+Streamline your pentesting and development workflows with Cobalt integrations.
+{{% /pageinfo %}}
+
+{{% alert title="Note" color="note" %}}
+The availability of this feature is based on your <a href="https://www.cobalt.io/pentest-pricing" target="_blank">PtaaS tier</a>.
+{{% /alert %}}
+ 
+Navigate to the **Integrations** page to get started.
+
+- Set up **Native** integrations in the Cobalt app to push Cobalt data to external apps. Here, you can also create [webhooks](/integrations/webhooks/) to get real-time pentest updates.
+- Enable **Partner** integrations in third-party apps. You need an [API token](/apiusecases/create_asset/#create-an-api-token-in-the-cobalt-ui) to pull Cobalt data to external apps.
+
+<!-- Links to Zendesk are temporary. We'll change them once the content is moved to Product Docs. -->
+| Integration | Type | Use |
+|:---|:---|:---|
+| <img src="/gsg/IntegrationsJira.png" width="30"> [Jira](https://cobaltio.zendesk.com/hc/en-us/sections/4407694113044-Integration-Guides) | Native | Push Cobalt findings as issues to Jira Cloud, Server, or Data Center. Synchronize Cobalt findings with Jira tickets bi-directionally.
```

Comment:
I may need to remove images. The first one doesn't display properly.

Reply:
If so, bummer. I'm glad that you're trying.